src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java

```diff
@@ -0,0 +1,88 @@
+package it.pagopa.interop.probing.caller.util;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Objects;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.Response;
+import it.pagopa.interop.probing.caller.config.client.RestClientConfig;
+import it.pagopa.interop.probing.caller.config.client.SoapClientConfig;
+import it.pagopa.interop.probing.caller.dto.EserviceContentDto;
+import it.pagopa.interop.probing.caller.dto.TelemetryDto;
+import it.pagopa.interop.probing.caller.dtos.Problem;
+import it.pagopa.interop.probing.caller.soap.probing.ObjectFactory;
+import it.pagopa.interop.probing.caller.soap.probing.ProbingResponse;
+import it.pagopa.interop.probing.caller.util.logging.Logger;
+
+@Component
+public class ClientUtil {
+
+  @Autowired
+  RestClientConfig restClientConfig;
+
+  @Autowired
+  SoapClientConfig soapClientConfig;
+
+  @Autowired
+  private Logger logger;
+
+  public TelemetryDto callProbing(EserviceContentDto service) {
+
+    TelemetryDto telemetryResult =
+        TelemetryDto.builder().eserviceRecordId(service.eserviceRecordId()).build();
+
+    logger.logMessageCallProbing(service.technology().getValue(), service.basePath()[0]);
+
+    long before = System.currentTimeMillis();
```

Comment:
this should be right before the call, and the calculation should be right after the call